### ISSUE TITLE
Warning with StrictMode

### ISSUE THREAD

**Issue by Silverwolf90** [jossmac/react-toast-notifications#67]
When using `<StrictMode>` the following appears:

```
backend.js:6 Warning: Legacy context API has been detected within a strict-mode tree.

The old API will be supported in all 16.x releases, but applications using it should migrate to the new version.

Please update the following components: Transition

Learn more about this warning here: https://fb.me/react-legacy-context
    in StrictMode
    in Unknown
```

**Comment by NeliHarbuzava** [comment]
I have this warning in console:
`Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: ToastProvider`

It seems that it related to this issue too. Versions:

```
"react": "^16.11.0",
"react-toast-notifications": "^1.5.0",
```

**Comment by jossmac** [comment]
@Silverwolf90 which version are you using?

Pretty sure [this lib is following the current best practice](https://github.com/jossmac/react-toast-notifications/blob/master/src/ToastProvider.js#L31) for React context.

**Comment by Silverwolf90** [comment]
Sorry I should have put more information. Currently using  `react@16.10.2` and `react-toast-notifications@2.2.5`.

But the issue is to do with the dependency on `react-transition-group`: https://github.com/reactjs/react-transition-group/issues/429

**Comment by jossmac** [comment]
Okay great, so this will be resolved by #70

**Comment by jossmac** [comment]
Available in `v2.2.6`
